frontend/pages/DashboardPage/cards/ActivityFeed/components/ScriptBatchSummaryModal/ScriptBatchSummaryModal.tsx

```diff
@@ -103,6 +104,15 @@ incompatible or cancelled."
       // for the TS compiler
       return null;
     }
+
+    let scriptsPath = paths.CONTROLS_SCRIPTS;
+    // null team id from API = no team
+    if (statusData.team_id === null) {
```

Comment:
I changed this to `0` in the backend

Reply:
Great!So we're in sync as of https://github.com/fleetdm/fleet/commit/5edb793652319ba6c42f1a09ad819e4601225183